gprotorch/dataloader/solvent_blacklist.txt

```diff
@@ -0,0 +1,30 @@
+09V
```

Comment:
I think it would be better to store this data in a list/dictionary in a dedicated file. [Example](https://github.com/a-r-j/graphein/blob/graphein-api/graphein/protein/resi_atoms.py)